pkg/transformer/openshift/openshift.go

```diff
@@ -346,7 +346,15 @@ func (o *OpenShift) Transform(komposeObject kobject.KomposeObject, opt kobject.C
 	for _, name := range sortedKeys {
 		service := komposeObject.ServiceConfigs[name]
 		var objects []runtime.Object
-
+		//replicas
+		var replica int
+		if opt.Replicas > 1 {
```

Comment:
Similar to above comment in regards to kubernetes.go, make the changes here as well!